src/responses/Albany_AbstractResponseFunction.hpp

```diff
@@ -34,6 +34,9 @@ namespace Albany {
     //! Get the vector space associated with this response.
     virtual Teuchos::RCP<const Thyra_VectorSpace> responseVectorSpace() const = 0;
 
+    //! Get the dual vector space associated with this response.
+    virtual Teuchos::RCP<const Thyra_VectorSpace> responseDualVectorSpace() const = 0;
```

Comment:
Is the responseDualVecSpace ever used?

Reply:
No, it is not. I forgot to remove that when working on the Thyra objects. I am removing it.